app/actions/rate-goal.js

```diff
@@ -12,4 +12,25 @@ export const onMoveSlider = value => {
   };
 };
 
+export const onBorderClick = goal => {
+  return {
+    type: types.BORDER_GOAL_CLICK,
+    goal: goal,
+  };
+};
+
+export const onDeleteGoal = goal => {
+  return {
+    type: types.DELETE_GOAL,
+    goal: goal,
+  };
+};
+
+export const onEditGoal = goal => {
+  return {
+    type: types.EDIT_GOAL,
+    goal: goal,
+  };
+};
+
```

Comment:
we have action creators called 'onEditGoal' and 'onDeleteGoal' now being exported from 2 files. This doesn't make much sense, as they all get put into the same object in main container anyway... I would keep them in just one of the 2 files...